tcl/org.testeditor.tcl.dsl.tests/src/org/testeditor/tcl/dsl/jvmmodel/MacroCallVariableResolverTest.xtend

```diff
@@ -22,29 +22,29 @@ class MacroCallVariableResolverTest extends AbstractUnmockedTclValidatorTest {
 	@Test
 	def void testResolvingReferenceToAssignmentVariable() {
 		// given
-		classUnderTest.macroUseStack = #[]
-		val variable = variableReference => [variable = assignmentVariable("some")]
+		classUnderTest.macroUseStack = #[] // empty call stack
+		val assignmentVariableRef = variableReference => [variable = assignmentVariable("some")]
 
 		// when
-		val resolvedVariable = classUnderTest.resolveVariableReference(variable)
+		val resolvedVariable = classUnderTest.resolveVariableReference(assignmentVariableRef)
 
 		// then
 		resolvedVariable.assertInstanceOf(VariableReference) //
-		.variable.assertInstanceOf(AssignmentVariable).name.assertEquals("some")
+		.variable.assertInstanceOf(AssignmentVariable).name.assertEquals(assignmentVariableRef.variable.name)
 	}
 
 	@Test
 	def void testResolvingReferenceToEnvironmentVariable() {
 		// given
-		classUnderTest.macroUseStack = #[]
-		val variable = variableReference => [variable = environmentVariables("some").head]
+		classUnderTest.macroUseStack = #[] // empty call stack
+		val environmentVariableRef = variableReference => [variable = environmentVariables("some").head]
 
 		// when
-		val resolvedVariable = classUnderTest.resolveVariableReference(variable)
+		val resolvedVariable = classUnderTest.resolveVariableReference(environmentVariableRef)
 
 		// then
 		resolvedVariable.assertInstanceOf(VariableReference) //
-		.variable.assertInstanceOf(EnvironmentVariable).name.assertEquals("some")
+		.variable.assertInstanceOf(EnvironmentVariable).name.assertEquals(environmentVariableRef.variable.name)
```

Comment:
The assertion is duplicated in this class. We could extract a small helper method here.